libcloud/common/abiquo.py

```diff
@@ -180,11 +180,12 @@ class AbiquoConnection(ConnectionUserAndKey, PollingConnection):
     responseCls = AbiquoResponse
 
     def __init__(self, user_id, key, secure=True, host=None, port=None,
-                 url=None, timeout=None):
+                 url=None, timeout=None, **kwargs):
```

Comment:
Would it be possible to explicitly declare all the possible arguments (https://libcloud.readthedocs.org/en/latest/development.html#don-t-abuse-kwargs)?

Would make it a lot more clear and obvious on what is going on (aka what arguments are expected).